docker/pipelines_runner.sh

```diff
@@ -117,6 +117,7 @@ function main {
   pipelines --project "${google_cloud_project}" run \
     --command "/opt/gcp_variant_transforms/bin/${command} ${df_required_args} ${df_optional_args}" \
     --output "${temp_location}"/runner_logs_$(date +%Y%m%d_%H%M%S).log \
+    --output-interval 60s \
```

Comment:
nit: I am not sure if this can have any negative impact on running the main task, specially because we are using a very small machine for this worker (`g1-small`).

What do you think about modifying it to 3 or 5 minutes intervals?

Reply:
I think your update to 200 seconds is fine. LGTM.

Reply:
Thanks @mbookman for your reply and also for your contribution.

I am merging this PR now.